commands/unarchive.py

```diff
@@ -1,16 +1,31 @@
+import os
 from argparse import ArgumentParser
 import shutil
 
+from helpers.cli import yes
 from helpers.fileutils import findProblemLocation
 from helpers.types import problemList
 from helpers.webutils import promptToFetch
 
 def unarchiveCommand(data):
+    folder = None
+    if data.get('temp'):
+        folder = ".temp/"
+    elif data.get('solved'):
+        folder = '.solved/'
```

Comment:
This feels a bit weird to me, and it's hard to pinpoint why. I think it might be the part where you denote the specific folder with a separate flag than `--all`, maybe that flag should just optionally take a folder name (and possibly insert the `.` before the name if not present)?

Reply:
Maybe the help-text could explain that the optional folder name could be `solved` or `temp`

Reply:
This should be fixed. There is now a --target flag which moves a problem to a specific folder, and --all moves all problems in current folder to a specific folder. a, s andt can be used as shortcut arguements for archive, solved and temp, and it is not required to add a . in front or a / behind your foldername.